scripts/httpd.coffee

```diff
@@ -40,3 +40,7 @@ module.exports = (robot) ->
   robot.router.get "/hubot/ip", (req, res) ->
     robot.http('http://ifconfig.me/ip').get() (err, r, body) ->
       res.end body
+
+  robot.router.post "/hubot/slack", (req, res) ->
```

Comment:
Please add some sort of authentication to this webhook. Otherwise, anyone can send messages to our slack by simply calling the API.

Reply:
Ok , will look into it